src/main/java/org/owasp/html/CssSchema.java

```diff
@@ -846,15 +846,15 @@ Property forKey(String propertyName) {
     builder.put("z-index", bottom);
     builder.put("repeating-linear-gradient()", linearGradient$Fun);
     builder.put("repeating-radial-gradient()", radialGradient$Fun);
-	DEFINITIONS = Map.copyOf(builder);
+    DEFINITIONS = Collections.unmodifiableMap(builder);
```

Comment:
Ok.  I see builder allocated locally at 263

Reply:
Exactly, I renamed all variables to include bulder to make it more clear.